lib/round.rb

```diff
@@ -0,0 +1,38 @@
+class Round
+  attr_reader :deck, :turns
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+    @number_correct = 0
+    @number_wrong = 0
+  end
```

Comment:
I like seeing that you recognize that you don't need an attr reader for every attribute you define -- that's a misconception I see a lot of students have at the beginning of mod 1, and looks like you are understanding that you only need an attr_Reader for an attribute when you want to read it outside of this class.